langchain4j-core/src/main/java/dev/langchain4j/store/memory/chat/ChatMemoryStore.java

```diff
@@ -12,7 +12,7 @@
  * Allows for flexibility in terms of where and how chat memory is stored.
  * <br>
  * <br>
- * Currently, the only implementation available is {@link InMemoryChatMemoryStore}.
+ * Currently, the only implementations available are {@link InMemoryChatMemoryStore} and {@link SingleSlotChatMemoryStore}.
```

Comment:
```suggestion
 * Currently, the only implementation available is {@link InMemoryChatMemoryStore}.
```
This does not compile as `SingleSlotChatMemoryStore` is in anoter module. Since `SingleSlotChatMemoryStore` is "internals", I guess it is ok to not mention it.